tests/unit/s2n_session_ticket_test.c

```diff
@@ -43,6 +43,28 @@ int mock_nanoseconds_since_epoch(void *data, uint64_t *nanoseconds)
     return 0;
 }
 
+uint8_t cb_session_id_data[S2N_TLS_SESSION_ID_MAX_LEN] = { 0 };
+size_t cb_session_id_len = 0;
+size_t cb_session_data_len = 0;
+uint32_t cb_session_lifetime = 0;
+static int s2n_test_session_ticket_callback(struct s2n_connection *conn,
+                                            uint8_t *session_id_data,
+                                            size_t session_id_len,
+                                            uint8_t *session_data,
+                                            size_t session_len,
+                                            uint32_t session_lifetime)
+{
+    /* Store the callback data for comparison at the end of the connection. The
+     * session_data itself is ignored because the data varies based on time. */
```

Comment:
We talked about this offline -- you should be able to override the wallclock implementation to test this.